schemas/manifest.cue

```diff
@@ -83,9 +83,18 @@
         "versions": [],
     },
     "kubernetes": {
-        "fileName": "",
+        "fileName": "kubernetes-node-linux-arch.tar.gz",
         "downloadLocation": "",
         "downloadURL": "https://acs-mirror.azureedge.net/kubernetes/v${PATCHED_KUBE_BINARY_VERSION}/binaries/kubernetes-node-linux-${CPU_ARCH}.tar.gz"
+        "versions": [ 
```

Comment:
suggest to expand the schema above, similar to `containerd_ver`, can add `kubernetes_ver ~= [0-9]+.[0-9]+.[0-9]+(-hotfix.[0-9]{8})?` (think this matches all versions)

Reply:
or not, if it's not actually a contract, just helps prevent manual errs.

Reply:
Updated it